src/Helpers/FunctionCallTagHandler/FunctionCallTagHandlerIOS.ts

```diff
@@ -22,9 +22,6 @@ const functionCallTagEventEmitter = new NativeEventEmitter(TagManagerBridge);
 const listeners: Array<Listener> = [];
 let listenerRegistered = false;
 
-// tagEventEmmiter is used to handle callbacks in JS
-let jsEventEmitter;
```

Comment:
Was this broken? Never see `jsEventEmitter` set

Reply:
Yes, it causes app to crash.